src/Cmfd.cpp

```diff
@@ -45,7 +45,7 @@ Cmfd::Cmfd() {
   _group_indices_map = NULL;
 
   /* Initialize boundaries to be reflective */
-  _boundaries = new boundaryType[4];
+  _boundaries = new int[4];
```

Comment:
Actually, this was just me being lazy and not wanting to deal with a circular header dependency that was wreaking havoc. I've dealt with this in my latest commit and reverted everything back to using the ``boundaryType`` enum.

Just for future reference, an enumeration is a distinct type whose value is restricted to one of several explicitly named constants. The values of the constants are values of an integral type known as the underlying type of the enumeration (*e.g.*, an unsigned int).